nautobot/docs/installation/nautobot.md

```diff
@@ -0,0 +1,255 @@
+
+If it helps, try to think of activating the virtualenv like entering a shell unique to Nautobot's application environment.
+
+```no-highlight
+$ source /opt/nautobot/bin/activate
+(nautobot) $
+```
+
+Observe that after activating, your prompt will now be preceded with the name of the virtualenv (`nautobot`).
+
+!!! note
+    From here on out, any time you see a command prompt preceded by `(nautobot)`, that is your indicator that you should be activated in the virtualenv.
+
+## Prepare the Virtual Environment
+
+Before we install anything into the virtualenv, we want to make sure that Pip is running the latest version. This is neecessary because sometimes when a new virtualenv is created, a cached version of Pip is installed into it. (Yes, even after we've deliberately upgraded Pip at the system level!)
+
+We also want to deliberately install the `wheel` library which will tell Pip to always try to install wheel packages if they are available. A [wheel is a pre-compiled Python package](https://realpython.com/python-wheels/), which is quicker and safer to install because it does not require development libraries or `gcc` to be installed on your system just so that some more advanced Python libraries can be compiled. 
+
+```no-highlight
+(nautobot) $ pip3 install --upgrade pip wheel
+```
+
+## Install Nautobot
+
+Use Pip to install Nautobot:
+
+```no-highlight
+(nautobot) $ pip3 install nautobot
+```
+
+Great! We have a virtualenv ready for use by the `nautobot` user, so let's proceed to verifying the installation.
+
+## Verify your Nautobot Installation
+
+You should now have a fancy `nautobot-server` command in your environment. This will be your gateway to all things Nautobot! Run it to confirm the installed version of `nautobot`:
+
+```no-highlight
+(nautobot) $ nautobot-server --version
+```
+
+## Configuration
+
+Before you can use Nautobot, you'll need to configure it by telling it where your database and Redis servers can be found, among other things. This is done with the `nautobot_config.py` configuration file.
+
+### Initialize your configuration
+
+Initialize a new configuration by running `nautobot-server init`. You may specify an alternate location and detailed instructions for this are covered in the documentation on [Nautobot Configuration](../../configuration).
+
+However, because we've set the `NAUTOBOT_ROOT`, this command will automatically create a new `nautobot_config.py` at the default location based on this at `/opt/nautobot/nautobot_config.py`:
+
+```no-highlight
+(nautobot) $ nautobot-server init
+```
+
+### Required Settings
+
+Your `nautobot_config.py` provides sane defaults for all of the configuration settings. You will inevitably need to update the settings for your environment.
+
+Prepare to edit `/opt/nautobot/nautobot_config.py`, and head over to the documentation on [Required Settings](../../configuration/required-settings) to tweak your required settings.
```

Comment:
Prepare to edit? ... I'm prepared now what

Reply:
Head over to required settings!